calico-enterprise/getting-started/install-on-clusters/kubernetes/helm.mdx

```diff
@@ -38,7 +33,226 @@ In this guide, you install the Tigera Calico operator and custom resource defini
+      type: Calico
+    calicoNetwork:
+      bgp: Disabled
+      ipPools:
+      - cidr: 10.244.0.0/16
+        encapsulation: VXLAN
+  EOF
+  ```
+
+1. Add any other customizations you require to `values.yaml`. To see values that can be customized in the chart, see the [helm docs](https://helm.sh/docs/) or run the following command:
+
+  ```bash
+  helm show values tigera-operator-{{version}}.tgz
+  ```
+
+   to see the values that can be customized in the chart.
+
+### Install {{prodname}}
+
+<Tabs>
+
+<TabItem label="Standalone" value="Standalone">
+
+1. [Configure a storage class for Calico Enterprise](../../../operations/logstorage/create-storage)
+
+1. Export the license key certificate and token.
+
+  ```bash
+  export LICENSE_KEY_CERTIFICATE=<your-license-key-certificate>
+  export LICENSE_KEY_TOKEN=<your-license-key-token>
+  ```
+
+1. Install the Tigera {{prodname}} operator and custom resource definitions using the Helm chart:
+
+  ```bash
+  helm install calico-enterprise tigera-operator-{{version}}.tgz \
+  --set-file imagePullSecrets.tigera-pull-secret=<path/to/pull/secret>,tigera-prometheus-operator.imagePullSecrets.tigera-pull-secret=<path/to/pull/secret> \
+  --set licenseKey.enabled=true \
+  --set licenseKey.certificate=$LICENSE_KEY_CERTIFICATE \
+  --set licenseKey.token=$LICENSE_KEY_TOKEN \
+  --namespace tigera-operator --create-namespace
+  ```
+
+   or if you created a `values.yaml` above:
+
+  ```bash
+  helm install calico-enterprise tigera-operator-{{version}}.tgz -f values.yaml \
+  --set-file imagePullSecrets.tigera-pull-secret=<path/to/pull/secret>,tigera-prometheus-operator.imagePullSecrets.tigera-pull-secret=<path/to/pull/secret> \
+  --set licenseKey.enabled=true \
+  --set licenseKey.certificate=$LICENSE_KEY_CERTIFICATE \
+  --set licenseKey.token=$LICENSE_KEY_TOKEN \
+  --namespace tigera-operator --create-namespace
+  ```
+
+1. You can now monitor progress with the following command:
+
+  ```bash
+  watch kubectl get tigerastatus
+  ```
+
+  Congratulations! You have now installed {{prodname}} using the Helm 3 chart.
+
+</TabItem>
+<TabItem label="Management" value="Management">
+
+1. [Configure a storage class for Calico Enterprise](../../../operations/logstorage/create-storage)
+
+1. Export the service port number, and the public IP or host of the management cluster. (Ex. "example.com:1234" or "10.0.0.10:1234".)
+
+  ```bash
+  export MANAGEMENT_CLUSTER_ADDR=<your-management-cluster-addr>
+  ```
+
+1. Export the service type (`NodePort` or `LoadBalancer`), the service port, target port and protocol.
+  
+  ```bash
+  export EXT_SERVICE_TYPE=<external-service-type>
+  export EXT_SERVICE_PORT=<external-service-port>
+  export EXT_SERVICE_TARGET_PORT=<external-service-target-port>
+  export EXT_SERVICE_PROTOCOL=<external-service-protocol>
+  ```
+  
+  If the external service is of type NodePort.
+
+  ```bash
+  export EXT_SERVICE_TYPE=<external-service-node-port>
+  ```
+
+1. Export 1..n managed clusters.
```

Comment:
I've not seen it this way before, but if more common for developers, ok. Or, 1-n.

Reply:
I've changed this to `one or more`